docs/cmb/build_process.md

```diff
@@ -78,93 +114,24 @@ ilab model evaluate --benchmark mmlu_branch --model ~/.local/share/checkpoints/h
 ilab model evaluate --benchmark mt_bench_branch --model ~/.local/share/checkpoints/hf_format/<checkpoint> --taxonomy-path ~/.local/share/instructlab/taxonomy --judge-model ~/.cache/instructlab/models/prometheus-8x7b-v2-0 --base-model ~/.cache/instructlab/models/granite-7b-redhat-lab --base-branch main --branch main
 ```
 
-## Hosting the release candidates
-
-!!! warning
-    This needs to be revisited as a process, this was a hack to start.
-
-rsync over the files
-```bash
-mkdir $(date +%F)
-cd $(date +%F)
-rsync --info=progress2 -avz -e <USERNAME>@<REMOTE>:~/.local/share/checkpoints/hf_format/samples_xxxxx ./
-```
-
-Set up (if needed)
-```bash
-python3.11 -m venv venv
-source venv/bin/activate
-pip install vllm
-./run.sh
-```
+## Publish to Huggingface
```

Comment:
This will be an `ilab` feature soon! https://github.com/instructlab/instructlab/pull/2768

Reply:
Nice! Get me `gguf` conversion too eh? ❤️